cloud/azure/postgresql/detectors-postgresql.tf

```diff
@@ -152,3 +152,33 @@ resource "signalfx_detector" "memory_usage" {
     parameterized_subject = "[{{ruleSeverity}}]{{{detectorName}}} {{{readableRule}}} ({{inputs.signal.value}}) on {{{dimensions}}}"
   }
 }
+
+resource "signalfx_detector" "serverlog_free_storage" {
+  name = "${join("", formatlist("[%s]", var.prefixes))}[${var.environment}] Azure PostgreSQL free storage"
+
+  program_text = <<-EOF
+        base_filter = filter('resource_type', 'Microsoft.DBforPostgreSQL/servers') and filter('primary_aggregation_type', 'true')
+        A = data('serverlog_storage_percent', filter=base_filter and ${module.filter-tags.filter_custom})${var.serverlog_free_storage_aggregation_function}
+        signal = (100-A).publish('signal')
+        detect(when(signal < threshold(${var.serverlog_free_storage_threshold_critical}), lasting="${var.serverlog_free_storage_timer}")).publish('CRIT')
```

Comment:
I know you follow the existing detector but I really recommend to use "greater" logic, this is what prefer the runners according to the poll we did.

and contrary to usual, this will simplify the signalflow !

Reply:
Will be done in another PR